docs/user/guides/data-exchange/consume-data.md

```diff
@@ -1,114 +1,8 @@
-## Data Exchange Tutorial
-
-Once the data space has been launched (please see [charts/umbrella/README.md](../../../charts/umbrella/README.md) until **Install --> Data Exchange Subset**), we will execute the following steps/calls to perform the communication between the components.
+# Consume Data
 
```

Comment:
It is important to note that in order to consume data it is necessary to have provided it before, that is, to “consume data” first. I know it is something logical, but I think it is good to point it out.
```suggestion
In order to be able to consume data, it is necessary to have previously [provided](./provide-data.md).
```